component/topology/syncer.go

```diff
@@ -31,13 +35,15 @@ var (
 type TopologySyncer struct {
 	do              *domain.Domain
 	topologySession *concurrency.Session
+	serverInfo      *ServerInfo
 	ctx             context.Context
 	cancel          context.CancelFunc
 }
 
 func NewTopologySyncer(do *domain.Domain) *TopologySyncer {
```

Comment:
What the hell is the "domain"??

Reply:
Currently, it is only used to maintain `pdClient` and `etcdClient` when `pd.endpoints` changed.